src/index.js

```diff
@@ -311,6 +311,25 @@ export default class KintoApi {
   }
 
   /**
+   * Retrieves information for a given collection.
+   *
+   * @param  {String} id      The collection name.
+   * @param  {Object} options The options object.
+   * @return {Promise<{Object}, Error>}
+   */
+  getCollection(id, options={}) {
+    const { bucket, headers } = {
```

Comment:
Sigh, now I realize I'm not merging the options anymore here as in other methods. I really need to homogenize options processing.